ToolChain/Debugger/Debugger.py

```diff
@@ -97,11 +99,38 @@ def __init__(self, inputFile=None,
             self.symbols = {}
             self.loadSymbols(symbolsFile=symbolsFile)
 
+        if breakpointFile != "" and breakpointFile is not None:
+            self.breakPoints = self.loadBreakpoints(breakpointFile)
+        else:
+            self.breakPoints =[]
+
         # At this point, debugging session is ready to be used
         self.debugLog("Debugging session is ready to be used. Have fun!")
         self.debug(inputFile=inputFile)
         self.tearDownLoggingFacilities()
 
+    def loadBreakpoints(self, breakpointFile):
+        """
+        This will simply load the breakpoints from file if -bp flag was provided.
```

Comment:
I would adjust the comment here. The -bp option relates to the calling code not to this part. This part is unaware (and does not need to be aware) that a -bp option exists.